ci/unit/test_main_node.sh

```diff
@@ -52,5 +52,14 @@ else
 
     if [ "$WITH_VALGRIND" == 'memcheck' ]; then
 	mv test_results/unit-test-*.memcheck.xml .
+	# Debugging
+	ls test_results || true
+	ls unit-test-*.memcheck.xml
+	ls unit-test-*.memcheck.xml | wc
```

Comment:
(lint) Use find instead of ls to better handle non-alphanumeric filenames. [SC2012]